flycs_sdk/pipelines.py

```diff
@@ -323,11 +322,20 @@ def _is_valid_start_time(start_time: Optional[datetime]) -> bool:
     if start_time is None:
         return True
 
-    if not isinstance(start_time, datetime):
-        raise TypeError("start_time must be a valid datetime object")
+    if not isinstance(start_time, datetime) and not isinstance(
+        start_time, pendulum.DateTime
+    ):
+        raise TypeError(
+            "start_time must be a valid datetime or pendulum.datetime object"
+        )
 
-    if start_time.tzinfo != timezone.utc:
-        raise ValueError("start_time timezone must be UTC")
+    if (
+        isinstance(start_time, pendulum.DateTime)
+        and start_time.timezone_name not in pytz.all_timezones
+    ):
+        raise ValueError(
+            "start_time timezone is invalid , please refers to https://en.wikipedia.org/wiki/List_of_tz_database_time_zones to see available time zones."
```

Comment:
maybe print the invalid timezone name as well so the user can clearly see where the issue is coming from ?